mappings/net/minecraft/block/Block.mapping

```diff
@@ -296,3 +301,6 @@ CLASS net/minecraft/unmapped/C_mmxmpdoq net/minecraft/block/Block
 			ARG 3 facing
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 o
+	CLASS C_znvnetlf
+		METHOD rehash (I)V
+			ARG 1 newN
```

Comment:
Is this supposed to have an `N` after it?

Reply:
I guess so, as in 'new number'. I should change it though

Reply:
Ah I see, something like `newNumber` or `newNo` would work

Reply:
The method is actually from `Object2ByteLinkedOpenHashMap`, and the parameter name comes from there

Reply:
Ah I see. Not something you're thinking of changing then?

Reply:
I changed it to `newSize`